packages/fxa-auth-server/lib/payments/subscription-reminders.ts

```diff
@@ -132,11 +132,32 @@ export class SubscriptionReminders {
         currentPeriodEnd: subscription.current_period_end,
         currentDateMs: Date.now(),
       });
+      const { email } = account;
+      const formattedSubscription = await this.stripeHelper.formatSubscriptionForEmail(
+        subscription
+      );
+      const {
+        amount,
+        currency,
+        interval_count,
+        interval,
+      } = await this.stripeHelper.findPlanById(
+        subscription.items.data[0].price.id
```

Comment:
This seems like a suboptimal way to get the plan ID. Given that I have the `subscription` object here with an expanded `customer`, is there a more direct way?

Reply:
Answering my own question here after a good night's sleep: In `sendReminders`, I have access to the `plan` for the `subscription` inside the `for` loop. So I could pass in `plan` or `plan_id` to `sendSubscriptionRenewalReminderEmail` as a second argument...